api/src/main/resources/marquez/db/migration/V19__alter_run_to_add_original_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE runs ADD original_name VARCHAR(255);
```

Comment:
Is there a description on the usage of `original_name`?

Reply:
OpenLineage uses strings as its run id. This is just for cataloging to retain the original name after we upconverted it to uuid.

The openlineage spec treats the run id as locally unique to the job which also raises some issues with backporting to the marquez API.

Reply:
Make sense. I would use `external_id` as the column name since it's an external run ID (_string_, _int_, etc) collected from another system.